docs/src/explanations/printing.md

```diff
@@ -61,11 +61,11 @@ printf(cf"myUInt = $myUInt%c") // myUInt = !
 
 There are special values you can include in your `cf` interpolated string:
 
-* `HierarchicalName` (`%m`): The hierarchical name of the signal
+* `HierarchicalModuleName` (`%m`): The hierarchical name of the current module
```

Comment:
Is `%m` in parentheses the right way to show this / does it hint that you can also use this? I.e., it might be clearer to indicate that this is lowered to `%m` in Verilog or something.

Reply:
`%m` is supported by Chisel--it's parsed into `HierarchicalModuleName`.